src/test/kotlin/com/terraformation/backend/customer/model/PermissionTest.kt

```diff
@@ -2068,6 +2129,7 @@ internal class PermissionTest : DatabaseTest() {
     private val uncheckedDraftPlantingSites = draftPlantingSiteIds.toMutableSet()
     private val uncheckedFacilities = facilityIds.toMutableSet()
     private val uncheckedModuleEvents = moduleEventIds.toMutableSet()
+    private val uncheckedModules = moduleIds.toMutableSet()
```

Comment:
Need to add a statement to `andNothingElse()` to verify that the permission is false for any IDs that weren't listed in an `expect()` call in a test case.